### PR TITLE
Feature - create Azure Storage metapackage

### LINKED ISSUE #67: Create metapackage: Arcus.Scripting.Storage
**Is your feature request related to a problem? Please describe.**
Create metapackage: Arcus.Scripting.Storage

**Describe the solution you'd like**
Create 1 package which imports all om the Arcus.Scripting.Storage.* modules.

### PR DESCRIPTION
**Pull request by stijnmoreels**
Creates new PS module project that imports all the Arcus Azure Storage PS module projects.

Closes #67

### CODE REVIEW

**Comment by stijnmoreels** [comment]
Is this enough to have a metapackage that includes all our Azure Storage PS modules, @mbraekman ?

**Comment by mbraekman** [comment]
> Is this enough to have a metapackage that includes all our Azure Storage PS modules, @mbraekman ?
> 
> Not sure though, if I should include the `.psm1` file here?

As you indicate, don't think a psm1-file should be included here, as this will simply be a copy/paste of the original psm1-files, forcing us to update it in multiple locations.
So this should be sufficient yes

File: docs/preview/features/powershell/azure-storage.md [hunk 1]
```diff
@@ -7,6 +7,14 @@ layout: default
 
 This module contains several sub-modules, all related to Azure Storage.
 
+## Installation
+
+To have access to the following sub-modules, you have to import the module:
```
Comment (mbraekman) [inline docs/preview/features/powershell/azure-storage.md:12]:
```suggestion
Install all Azure Storage-related modules at once via:
```

**Comment by mbraekman** [comment]
@stijnmoreels , related to the `Arcus.Scripting.All` module, should this then be renamed to `Arcus.Scripting.Storage.All`

**Comment by stijnmoreels** [comment]
> @stijnmoreels , related to the `Arcus.Scripting.All` module, should this then be renamed to `Arcus.Scripting.Storage.All`?

Aha, yes, good catch and suggestion! 👍

**Review by mbraekman — APPROVED** [review]
Looking good!

**Pull request merged**